src/commands/mount.rs

```diff
@@ -31,35 +32,53 @@ pub struct MountCmd {
     time_template: Option<String>,
 
     /// Don't allow other users to access the mount point
-    #[clap(long)]
+    #[clap(short, long)]
     #[merge(strategy=conflate::bool::overwrite_false)]
-    no_allow_other: bool,
+    exclusive: bool,
 
     /// How to handle access to files. [default: "forbidden" for hot/cold repositories, else "read"]
     #[clap(long)]
     #[merge(strategy=conflate::option::overwrite_none)]
-    file_access: Option<String>,
+    file_access: Option<FilePolicy>,
 
     /// The mount point to use
     #[clap(value_name = "PATH")]
     #[merge(strategy=conflate::option::overwrite_none)]
-    mountpoint: Option<PathBuf>,
+    mount_point: Option<PathBuf>,
 
     /// Specify directly which snapshot/path to mount
     #[clap(value_name = "SNAPSHOT[:PATH]")]
     #[merge(strategy=conflate::option::overwrite_none)]
     snapshot_path: Option<String>,
+
+    /// Other options to use for mount
+    #[clap(short, long = "option", value_name = "OPTION")]
+    #[merge(strategy = conflate::vec::overwrite_empty)]
+    options: Vec<String>,
 }
 
 impl Override<RusticConfig> for MountCmd {
     // Process the given command line options, overriding settings from
     // a configuration file using explicit flags taken from command-line
     // arguments.
     fn override_config(&self, mut config: RusticConfig) -> Result<RusticConfig, FrameworkError> {
-        let mut self_config = self.clone();
-        // merge "mount" section from config file, if given
-        self_config.merge(config.mount);
+        // Merge by precedence, cli <- config <- default
+        let mut self_config = self.clone().merge_precedence(config.mount, Self::new());
```

Comment:
I just wonder if `self.clone().merge_from(config.mount).merge_from(Self::new())` would not be even more clear?!?